CONTRIBUTING.md

```diff
@@ -259,37 +342,36 @@ The project contains [.editorconfig](https://github.com/chocolatey-community/cho
 
 Keep the package source files clean and remove obsolete or outdated code and unnecessary comments. Comment non-obvious code so that others can easily understand what it does.
 
-# 3. Teamwork
+### Update Script
+
+An update script are the PowerShell scripts responsible for getting the information about what the most current version of the software is, where it can be acquired from and is typically called `update.ps1`.
+These scripts should be located in the root of the package directory, right next to the metadata (`.nuspec`) file.
+The update script should be compatible with PowerShell v5, but may not need to be compatible with PowerShell Core.
 
-Good communication is essential so please take a look at [Code of Conduct](https://github.com/chocolatey/.github/blob/main/CODE_OF_CONDUCT.md) regarding it.
+#### Use `UseBasicParsing`
 
-### 3.1 Update CODEOWNERS
+If you use the PowerShell cmdlet in the update script, always make sure to also pass in `-UseBasicParsing` to this script. There may be cases where using this parameter is not possible, in these cases a comment right before the call should be specified about why `-UseBasicParsing` can not be used.
```

Comment:
A discussion for another time:
Is there a reason we wouldn't _enforce_ UseBasicParsing by use of global variables and default parameters, and ensure that packages that do not require it _must specify it_ as `-UseBasicParsing:$false`? (Other than it being a bizarre pattern)

Reply:
I didn't know you could enforce it. Also, I think it can only be enforced partially (since each script is launched in its own job, I have sometimes seen that configured variables aren't carried over).